### PR TITLE
OPAL-2880 - replace new plugin's site.properties file by old plugin's…

### PR DESCRIPTION
**Pull request by meek0**
… if present

### CODE REVIEW

File: jennite-vcf-store/src/main/dist/deb/debian/postinst [hunk 1]
```diff
@@ -14,10 +14,19 @@ case "$1" in
...
         if [ -d "$OPAL_HOME"/plugins ]; then
+        	OLD_PLUGIN=$(ls -t "$OPAL_HOME"/plugins/ | grep jennite-vcf-store | head -1)
+
           cp -r /usr/share/jennite* $OPAL_HOME/plugins
+
+          if [ ! -z "$OLD_PLUGIN" ]; then
+          	echo "Copying $OLD_PLUGIN/site.properties to new installation."
+						NEW_PLUGIN=$(ls -t $OPAL_HOME/plugins/ | grep jennite-vcf-store | head -1)
+						cp $OPAL_HOME/plugins/$OLD_PLUGIN/site.properties $OPAL_HOME/plugins/$NEW_PLUGIN/
+					fi
```
Comment (ymarcon) [inline jennite-vcf-store/src/main/dist/deb/debian/postinst]: Please fix code indentation

**Review by ymarcon — CHANGES_REQUESTED** [review]

File: jennite-vcf-store/src/main/dist/deb/debian/postinst [hunk 2]
```diff
@@ -14,10 +14,19 @@ case "$1" in
         . /etc/default/opal
         mkdir -p $OPAL_HOME/plugins
         if [ -d "$OPAL_HOME"/plugins ]; then
+        	OLD_PLUGIN=$(ls -t "$OPAL_HOME"/plugins/ | grep jennite-vcf-store | head -1)
+
           cp -r /usr/share/jennite* $OPAL_HOME/plugins
+
+          if [ ! -z "$OLD_PLUGIN" ]; then
```
Comment (ymarcon) [inline jennite-vcf-store/src/main/dist/deb/debian/postinst]: Maybe you could also check for `site.properties` file existence in `$OLD_PLUGIN` folder as this is the file you will copy?

File: jennite-vcf-store/src/main/dist/deb/debian/postinst [hunk 3]
```diff
@@ -14,10 +14,19 @@ case "$1" in
...
+          	echo "Copying $OLD_PLUGIN/site.properties to new installation."
+						NEW_PLUGIN=$(ls -t $OPAL_HOME/plugins/ | grep jennite-vcf-store | head -1)
+						cp $OPAL_HOME/plugins/$OLD_PLUGIN/site.properties $OPAL_HOME/plugins/$NEW_PLUGIN/
```
Comment (ymarcon) [inline jennite-vcf-store/src/main/dist/deb/debian/postinst]: You could backup the original `site.properties` to `site.properties.pkg` before copying the old one.

Comment (ymarcon) [reply jennite-vcf-store/src/main/dist/deb/debian/postinst]: Forget that, what's in the plugins folder is a copy of `/usr/share/...` anyway.

**Comment by meek0** [comment]
Changes are done.

**Pull request merged**